client/src/components/tabs/cart.js

```diff
@@ -32,7 +32,7 @@ const Cart = () => {
       method: 'DELETE',
     })
     await response.json();
-    const deleteItemFunction = cart.filter((clothes) => clothes.items_id.id !==handleDeleteShirt);
+    const deleteItemFunction = cart.filter((clothes) => clothes.items_id !==handleDeleteShirt);
```

Comment:
variables named with `handleX` are usually handler functions so a different name might be more clear